Source/WebCore/testing/InternalSettings.cpp

```diff
@@ -280,6 +284,18 @@ ExceptionOr<void> InternalSettings::setPictographFontFamily(const String& family
     return { };
 }
 
+ExceptionOr<void> InternalSettings::setMathFontFamily(const String& family, const String& script)
+{
+    if (!m_page)
+        return Exception { ExceptionCode::InvalidAccessError };
+    UScriptCode code = scriptNameToCode(script);
+    if (code == USCRIPT_INVALID_CODE)
+        return { };
```

Comment:
Maybe I am missing something here; but instead of silently doing nothing, wouldn't it be better to do an `Exception`-return when the passed argument is an invalid script name?

Reply:
Right, the other font families indeed do the same. On second thought, I suppose it doesn't matter much that the case is a no-op, because then the font is not really changed and that would make tests fail because the expectations would not be matched 🤔